website/aws.erb

```diff
@@ -203,6 +203,9 @@
                         <li<%= sidebar_current("docs-aws-datasource-security-group") %>>
```

Comment:
We don't expect most folks to know this, but an odd side effect of having the singular and plural data sources is that both entries will be highlighted in the sidebar if you are on the singular data source page since the `sidebar_current` prefix matches both. Our current convention is to go back to the singular data source and append `-x` to the `sidebar_current` in both the sidebar and singular data source page.